jwst/datamodels/model_base.py

```diff
@@ -948,12 +948,6 @@ def protected_keyword(path):
             if not protected_keyword(path):
                 set_hdu_keyword(self._instance, d, path)
 
-        # Perform updates to extra_fits area of a model
```

Comment:
I would really like to see this controlled by an optional argument, rather than just eliminating the capability.